src/main/java/com/airlines/dao/impl/AirplaneDaoImpl.java

```diff
@@ -75,18 +97,21 @@ public void save(Airplane airplane) {
 
     @Override
     public Airplane findOne(String codeName) {
-        Airplane airplane = new Airplane();
+        Airplane airplane = null;
         try (Connection connection = dataSource.getConnection();
-             PreparedStatement selectByIdStatement = connection.prepareStatement(SELECT_BY_CODE_NAME_SQL);) {
-            selectByIdStatement.setString(1, codeName);
-            ResultSet resultSet = selectByIdStatement.executeQuery();
+             PreparedStatement selectByCodeNameStatement = connection.prepareStatement(SELECT_BY_CODE_NAME_SQL);) {
+            selectByCodeNameStatement.setString(1, codeName);
+            ResultSet resultSet = selectByCodeNameStatement.executeQuery();
```

Comment:
How this change connected with your task?
You could change this DaoImpl only in part where you change old entity style (constructor + setters) to new one (builder).
Refactoring whole project is not part of story
Discard this change

Reply:
done in https://github.com/Matsukata/airlines-jdbc-app/pull/5/commits/625cab1de0c97028d188cc5a80294e281f46b375